api/src/test/java/org/pmiops/workbench/api/DiskAdminControllerTest.java

```diff
@@ -83,4 +84,18 @@ public void listDisksInWorkspace_workspaceNotFound() {
     assertThrows(
         NotFoundException.class, () -> diskAdminController.listDisksInWorkspace(WORKSPACE_NS));
   }
+
+  @Test
+  public void deleteDisk() {
```

Comment:
Not sure if we are doing this yet, should we have a test to check how service handles exception?

Reply:
Ahh i now see a test listDisksInWorkspace_workspaceNotFound, we should add some negative test for deletedisk() or if leo threw an error while deleting the disk maybe unauthorized?

Reply:
Added tests